examples/lock-app/nrfconnect/README.md

```diff
@@ -63,43 +111,29 @@ states are depicted:
     provisioned, but does not yet have full network (Thread) or service
     connectivity.
 
-*   _Solid On_ &mdash; The device is fully provisioned and has full network and
-    service connectivity.
-
-**Button #1** can be used to initiate a OTA software update as well as to reset
-the device to a default state.
-
-Pressing and holding Button #1 for 6 seconds initiates a factory reset. After an
```

Comment:
I think we should keep the factory reset procedure description in some form.

Reply:
Actually I think that it's not the right place for this and it should be located in some testing section or tutorial. If we will keep it here we will separate some testing actions from the other and user will have to jump between it.